spec/classes/azure_spec.rb

```diff
@@ -0,0 +1,206 @@
+require 'spec_helper'
+
+describe 'azure' do
```

Comment:
Mixed up a couple different sayings here but you get the point...  Apparently I'm not the first to make that mistake though: http://languagelog.ldc.upenn.edu/nll/?p=727